projects/notebooks/roles/notebooks_dashboard_scale_test/tasks/artifacts_driver.yml

```diff
@@ -59,7 +59,9 @@
     oc -n {{ notebooks_dashboard_scale_test_minio_namespace }} -c ubi8 \
        rsync "{{ minio_podname_cmd.stdout }}:/artifacts/to_export/" "{{ artifact_extra_logs_dir }}" --quiet
 
-    rm -f "{{ artifact_extra_logs_dir }}/os-release"
+    rm -f "{{ artifact_extra_logs_dir }}/os-release" \
+    find "{{ artifact_extra_logs_dir }}/" -type d -name "Standard Data Science Pipeline*" -exec rm -rf {} + \
+    rm -rf "{{ artifact_extra_logs_dir }}/pipelines"
```

Comment:
you need to remove the `\` at the end of the string, it won't work with it I think